.github/workflows/go-test-config/action.yaml

```diff
@@ -116,6 +116,12 @@ runs:
         tests/github-action-helper.sh create_stale_subvolume
         subVol=$(kubectl rook-ceph ${NS_OPT} subvolume ls --stale | awk '{print $2}' | grep csi-vol)
         kubectl rook_ceph ${NS_OPT} subvolume delete myfs $subVol
+      
+    - name: Get rbd list
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        set -ex
+        kubectl rook-ceph ${NS_OPT} rbd ls
```

Comment:
As a follow up, could a validation be added to this test to confirm the expected volume(s) are listed? Otherwise, we might have a regression in the future.

Reply:
@travisn I will add tests

Reply:
Hi @travisn I opened two PRs for this test. I added a sleep statement because of some issues I observed on my local machine.
https://github.com/rook/kubectl-rook-ceph/pull/342
https://github.com/rook/rook/pull/15157